library/spdm_responder_lib/libspdm_rsp_capabilities.c

```diff
@@ -157,6 +157,11 @@ return_status spdm_get_response_capabilities(IN void *context,
     spdm_context = context;
     spdm_request = request;
 
+    if (spdm_request->header.spdm_version != spdm_get_connection_version(spdm_context)) {
```

Comment:
How does the code work?

The responder need use this spdm_version to set connection version.